tensorflow/workspace2.bzl

```diff
@@ -822,22 +822,22 @@ def _tf_repositories():
     # https://github.com/bazelbuild/rules_apple/releases
     tf_http_archive(
         name = "build_bazel_rules_apple",
-        sha256 = "36072d4f3614d309d6a703da0dfe48684ec4c65a89611aeb9590b45af7a3e592",
```

Comment:
Just fyi if this change introduces any regression, we'll need to roll this back.

Reply:
Sounds good!